docs/changes/units/17059.feature.rst

```diff
@@ -0,0 +1 @@
+Added a ``np.arange`` dispatch for ``Quantity``.
```

Comment:
Maybe expand
```
Added a ``np.arange`` dispatch for ``Quantity`` (requires one to use ``like=<some_quantity>``).
```
Just to avoid people getting too excited...